ruff.toml

```diff
@@ -1,5 +1,4 @@
 exclude = ["ribasim_qgis/tomllib/*"]
-extend-include = ["*.ipynb"]
```

Comment:
This has been the default since ruff 0.6: https://astral.sh/blog/ruff-v0.6.0#jupyter-notebooks-are-now-linted-and-formatted-by-default